plenum/server/view_change/view_changer.py

```diff
@@ -525,7 +525,11 @@ def _start_selection(self):
         if self.view_change_in_progress:
             self.view_change_in_progress = False
             self.node.on_view_change_complete()
-            self.instanceChanges.pop(self.view_no - 1, None)
+            # when we had INSTANCE_CHANGE message, they added into instanceChanges
+            # by msg.view_no. When view change was occured and view_no is changed,
+            # then we should delete all INSTANCE_CHANGE messages with current (already changed)
+            # view_no (which used in corresponded INSTANCE_CHANGE messages)
+            self.instanceChanges.pop(self.view_no, None)
```

Comment:
I think we can remove all instance changes with `view_no <= self.view_no` as  we do for `self._next_view_indications`